pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/util/LiquibaseUtil.java

```diff
@@ -205,7 +205,7 @@ private void runMigrations(String changelogFile, String context) throws Liquibas
                 liquibase.update(new Contexts(context));
             } catch (LiquibaseException originalError) {
                 log.error("LiquibaseException: {}", originalError.getMessage());
-                if (tag != null) {
+                if (tag != null && originalError.getCause() != null) {
```

Comment:
While testing the error handling for liquibase locks, I discovered this NPE.  The fix is to only log this addition message if `getCause()` is not null.